src/style.css

```diff
@@ -0,0 +1,139 @@
+* {
+
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+
+  font-family: "Avenir", sans-serif;
```

Comment:
❗❗ hhey, so did you import the font before using it?
hint: `@font-face {` font wouldn't apply without it